README.md

```diff
@@ -159,6 +162,19 @@ kong_nginx_http_current_connections{state="reading"} 0
 kong_nginx_http_current_connections{state="total"} 8
 kong_nginx_http_current_connections{state="waiting"} 0
 kong_nginx_http_current_connections{state="writing"} 1
+# HELP kong_memory_lua_shared_dict_bytes Allocated slabs in bytes in a shared_dict
+# TYPE kong_memory_lua_shared_dict_bytes gauge
+kong_memory_lua_shared_dict_bytes{shared_dict="kong",kong_subsystem="http"} 40960
+.
+.
+# HELP kong_memory_lua_shared_dict_total_bytes Total capacity in bytes of a shared_dict
+# TYPE kong_memory_lua_shared_dict_total_bytes gauge
+kong_memory_lua_shared_dict_total_bytes{shared_dict="kong",kong_subsystem="http"} 5242880
```

Comment:
This metric seems to be different from the rest of the others. All the others are usage metrics but this one seems like a capacity metric.
Should this instead be named as `kong_memory_lua_shared_dict_capacity_bytes`?

Reply:
@hbagdi This is indeed a capacity metric, but it's not newly added in this PR.
I add it into the Readme because it seems got missed before. Consider its
age in the code we should probably keep with the old naming?

Reply:
Ah, correct. That would be a breaking change.